region/brinstar/pink/Dachora Room.json

```diff
@@ -231,6 +248,28 @@
         {"refill": ["Energy", "Missile", "Super"]}
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "Blockade Moondance",
+      "notable": false,
+      "requires": [
+        "h_canUseMorphBombs",
+        {"or": [
+          {"and": [
+            "canMoondance",
+            "SpeedBooster"
+          ]},
+          {"and": [
+            "ScrewAttack",
+            "canExtendedMoondance"
+          ]}
+        ]}
+      ],
+      "note": [
+        "Break exactly the bottom-right and top-right Bomb Blocks and Unmorph while on the top block hole, to begin the Moondance.",
+        "Clear a path to the left in advance, and clear all enemies."
```

Comment:
```suggestion
        "Clear all the enemies before beginning."
```
It could be helpful to give some explanation of how Screw Attack comes into play